api/src/api/userRouter.test.ts

```diff
@@ -343,6 +345,186 @@ describe("userRouter", () => {
         expect(stubTimeStampBusinessSearch.search).toHaveBeenCalledWith(userData.profileData.businessName);
         expect(stubUpdateOperatingPhase).toHaveBeenCalledWith(userData);
       });
+
+      describe("when businessPersona is 'FOREIGN'", () => {
+        it("does not update business name search if dbaBusinessNameAvailability is undefined", async () => {
```

Comment:
this is only true if `businessNameSearch` is ALSO undefined right? 
If `businessNameSearch` was defined and `dbaBusinessNameAvailability` was undefined then it WOULD do a search, right?

Reply:
`businessNameSearch` is in reference to the object that used to contain `businessNameAvailability`, correct?

Yes, I can make this more explicit. If neither `businessNameAvailability` or `dbaBusinessNameAvailability` are defined, the search will not happen.